src/main/java/frc/robot/subsystems/AimingSubsystem.java

```diff
@@ -64,28 +68,40 @@ public AimingSubsystem() {
     mChooser.addOption("Brake", AimingMotorMode.BRAKE);
     mChooser.addOption("Coast", AimingMotorMode.COAST);
     mChooser.setDefaultOption("Brake", AimingConstants.INITIAL_MOTOR_MODE);
+    SmartDashboard.putData("Elevator Motor Mode", mChooser);
+
+    mLeftElevatorEncoder = mLeftElevatorMotor.getEncoder();
+    mRightElevatorEncoder = mRightElevatorMotor.getEncoder();
+
+    configureDevices();
   }
 
   // Setting Conversions and Inversions
   public void configureDevices() {
 
     //initialize PID Controller Constants for SlotConfigs
-
     mElevatorControllerSlot0Configs = AimingConstants.mElevatorPIDConstants.toTalonConfiguration().Slot0;
-
-    mLeftElevatorMotor.getConfigurator().apply(mElevatorControllerSlot0Configs);
-    mRightElevatorMotor.getConfigurator().apply(mElevatorControllerSlot0Configs);
     
     mWristController = AimingConstants.mWristPIDConstants.toWPIController();
     //note: configuration uses internal encoders inside the motors, subject to change
   
-    mLeftElevatorMotor.setInverted(true);
+    mLeftElevatorMotor.setInverted(AimingConstants.ELEVATOR_LEFT_IS_INVERTED);
     mLeftWristMotor.setInverted(true);
     mWristController.setTolerance(AimingConstants.WRIST_TOLERANCE_DEG);
 
```

Comment:
Set the elevatorController Tolerance too

Reply:
Done.